packages/modelviewer.dev/examples/stagingandcameras/index.html

```diff
@@ -290,22 +290,26 @@ <h4>Event listeners can cooperate with camera-controls</h4>
   }, true);
 
   modelViewer.addEventListener('touchmove', (event) => {
-    if (!panning || event.touches.length !== 2)
+    if (!panning || event.targetTouches.length !== 2)
       return;
 
-    const {touches} = event;
-    const thisX = 0.5 * (touches[0].clientX + touches[1].clientX);
-    const thisY = 0.5 * (touches[0].clientY + touches[1].clientY);
+    const {targetTouches} = event;
+    const thisX = 0.5 * (targetTouches[0].clientX + targetTouches[1].clientX);
+    const thisY = 0.5 * (targetTouches[0].clientY + targetTouches[1].clientY);
     movePan(thisX, thisY);
   }, true);
 
   self.addEventListener('mouseup', (event) => {
     recenter(event);
   }, true);
   
-  self.addEventListener('touchend', (event) => {
-    if (event.touches.length === 0) {
+  modelViewer.addEventListener('touchend', (event) => {
+    if (event.targetTouches.length === 0) {
       recenter(event.changedTouches[0]);
+
+      if (event.cancelable) {
+        event.preventDefault();
```

Comment:
Prevents emulation of `mouseup` event which was invoking `recenter` a second time.